iceoryx_hoofs/include/iceoryx_hoofs/cxx/unique_ptr.hpp

```diff
@@ -31,7 +31,7 @@ namespace cxx
 /// instances with different deleters to be stored in the same containers.
 ///
 template <typename T>
-class unique_ptr
+class unique_ptr final
```

Comment:
@elBoberido I really think we should not use the final rule. I can elaborate if needed.

Reply:
@MatthiasKillat We agreed to keep it, closing this thread.